### PR TITLE
Add support for <sub-title> (episode name) XMLTV tags

### LINKED ISSUE #91: EpisodeName Support
Can you add support for EpisodeName via the xmltv file?

The value is in the sub-title field.

I was able to add support by doing the following in PVRIptvData.cpp
```
@@ -274,6 +274,7 @@
     GetNodeValue(pChannelNode, "title", entry.strTitle);
     GetNodeValue(pChannelNode, "desc", entry.strPlot);
     GetNodeValue(pChannelNode, "category", entry.strGenreString);
+    GetNodeValue(pChannelNode, "sub-title", entry.strEpisodeName);
 
     xml_node<> *pIconNode = pChannelNode->first_node("icon");
     if (pIconNode == NULL || !GetAttributeValue(pIconNode, "src", entry.strIconPath))
@@ -715,7 +716,7 @@
       tag.iSeriesNumber       = 0;     /* not supported */
       tag.iEpisodeNumber      = 0;     /* not supported */
       tag.iEpisodePartNumber  = 0;     /* not supported */
-      tag.strEpisodeName      = NULL;  /* not supported */
+      tag.strEpisodeName      = myTag->strEpisodeName.c_str();
       tag.iFlags              = EPG_TAG_FLAG_UNDEFINED;
```

And in PVRIptvData.h
```
@@ -41,6 +41,7 @@
   std::string strPlot;
   std::string strIconPath;
   std::string strGenreString;
+  std::string strEpisodeName;
 };
 
 struct PVRIptvEpgChannel
```

### PR DESCRIPTION
**Pull request by Jalle19**
Closes #91

@excorp can you try this?

### CODE REVIEW

**Comment by Jalle19** [comment]
ping @excorp

**Comment by shaneypoopers** [comment]
Did this get approved yet? I don't see it in the main branch?

**Comment by kyl416** [comment]
Any update on getting this approved?

File: src/PVRIptvData.cpp [hunk 1]
```diff
@@ -1055,51 +1056,6 @@ void PVRIptvData::ApplyChannelsLogosFromEPG()
     PVR->TriggerChannelUpdate();
 }
 
-void PVRIptvData::ReaplyChannelsLogos(const char * strNewPath)
```
Comment (afedchin) [inline src/PVRIptvData.cpp:1058]: how this removal related to PR changes?

**Review by afedchin — CHANGES_REQUESTED** [review]
please drop unrelated changes.

**Comment by phunkyfish** [comment]
Closing as superseded by #261 and #263

**Pull request closed without merging**